Geometry/HGCalCommonData/data/v1/hgcalEE.xml

```diff
@@ -110,6 +110,7 @@
   <SpecPar name="HGCalEE">
     <PartSelector path="//HGCalEESensitive.*"/>
     <Parameter name="Volume" value="HGCalEESensitive" eval="false"/>
+    <Parameter name="GeometryMode" value="HGCalGeometryMode::Square" eval="false"/>

```

Comment:
@davidlange6 - your assumption is not correct. HGCal "GeometryMode" is used starting from this PR. It is either a square or a hexagon reflecting its sensors shape.